index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>ADOPTADOG</title>
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+  </head>
+
+  <body>
+      <header>
+        <img src="images/cart-icon.svg" alt="Shopping cart icon." width="1">
+        <button>Checkout</button>
+        <img src="images/adoptadog-logo.svg" alt="Profile of a brown dog, staring at something." width="1">
+        <h1>ADOPTADOG</h1>
+        <h3>Seattle's premier dog adoption agency.</h3>
+      </header>
+
+      <nav>
+        <ul>
+          <li><a href="index.html" target="_blank">Home</a>|</li>
+          <li><a href="dogs.html" target="_blank">Dogs</a>|</li>
+          <li><a href="blog.html" target="_blank">Blog</a>|</li>
+        </ul>
+      </nav>
+
+      <section>
+          <img src="images/home-intro.jpg" alt="Picture of a man hugging a smiling golden retriever." width="1">
+          <h4>Adopt a Dog</h4>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+          </p>
+      </section>
+
+      <section class="flexbox-container">
+        <h3>W3 wanted me to add a header to my section.</h3>
```

Comment:
Yeah, it can be annoying. If this is something that W3 is picky about, and if you don't want a header you can just swap it to a div.